brainstormforce/astra
# AST-1315 - Added new design options for dark mode switcher

### Description
 Design options for dark mode switcher element

### Screenshots
- https://share.bsf.io/rRuWNvzZ

### Types of changes
- New feature - non-breaking change 

### How has this been tested?
- Check every option 
- With light mode
- with dark mode
- Check on frontend
- Check with customizer preview

### Checklist:
